Ghidra/Processors/RISCV/data/languages/riscv.rvv.sinc

```diff
@@ -3393,7 +3394,7 @@ define pcodeop vsetvli_e64m1tumu;
 :vsetvli   rd, rs1, "e32,m1,tu,ma"    is op2030=0x090 & op3131=0x0 & rs1 & op1214=0x7 & rd & op0006=0x57 {rd=vsetvli_e32m1tuma(rs1);}
 :vsetvli   rd, rs1, "e32,mf2,tu,ma"   is op2030=0x097 & op3131=0x0 & rs1 & op1214=0x7 & rd & op0006=0x57 {rd=vsetvli_e32mf2tuma(rs1);}
 :vsetvli   rd, rs1, "e64,m1,tu,ma"    is op2030=0x098 & op3131=0x0 & rs1 & op1214=0x7 & rd & op0006=0x57 {rd=vsetvli_e64m1tuma(rs1);}
-:vsetvli   rd, rs1, "e32,mf8,tu,ma"   is op2030=0x09d & op3131=0x0 & rs1 & op1214=0x7 & rd & op0006=0x57 {rd=vsetvli_e64mf8tuma(rs1);}
+:vsetvli   rd, rs1, "e32,mf8,tu,ma"   is op2030=0x09d & op3131=0x0 & rs1 & op1214=0x7 & rd & op0006=0x57 {rd=vsetvli_e32mf8tuma(rs1);}
```

Comment:
missing "define pcodeop vsetvli_e32mf8tuma" leads to compile errors:
```
> Task :RISCV:sleighCompile
Compiling ./data/languages/riscv.ilp32d.slaspec:
riscv.rvv.sinc:3397: unknown macro, userop, or specific symbol 'vsetvli_e32mf8tuma' in macro, user operation, or subpiece application
```

Reply:
Thanks!  This should now be fixed with ffb5411.

Any comments on other vset* patterns that might be found in common exemplars would be appreciated.  I'm grinding through autovectorized vector instruction sequences generated from https://github.com/ggerganov/llama.cpp currently, and will likely try and make the sinc pcodeop names align a bit better with the gcc riscv intrinsics defined in https://github.com/riscv-non-isa/rvv-intrinsic-doc.